keyboards/thedogkeyboard/matrix.c

```diff
@@ -0,0 +1,287 @@
+/*
```

Comment:
What's the reason for this custom matrix? It looks pretty identical to the core matrix.c.

Reply:
I was there for  COL2ROW2COL.
![SnapCrab_NoName_2019-11-6_13-17-20_No-00](https://user-images.githubusercontent.com/15257475/69113624-de717000-0ac6-11ea-8f0b-d603c023e05b.png)